src/main/java/nextstep/subway/domain/Section.java

```diff
@@ -0,0 +1,96 @@
+package nextstep.subway.domain;
+
+import javax.persistence.*;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+@Entity
+@Table(name = "SECTION")
+public class Section extends BaseEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "up_station_id")
+    private Station upStation;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "down_station_id")
+    private Station downStation;
+    private Integer distance;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "line_id")
+    private Line line;
+
+    protected Section() {
```

Comment:
```suggestion
    protected Section() {}
```
[컨벤션](https://naver.github.io/hackday-conventions-java/#permit-concise-empty-block)에 따르면 이렇게 써도 될 것 같아요..!